src/js/tabs/vtx.js

```diff
@@ -891,7 +891,7 @@ vtx.initialize = function (callback) {
                 $("#save_button").html(oldText);
             }, 2000);
 
-            TABS.vtx.initialize();
+            setTimeout(() => TABS.vtx.initialize, 3000); // Allow firmware to make relevant changes before initialization
```

Comment:
What about to include it in in the setTimeout for the save button that is just before and increase the timeout of it? At least in this way, you know that the tab will be refreshed at the same time than the button, seems more "solid" for me. 

Another solution a little more ellaborated is to use more button messages:
- The button contains "Save". The users pushes it.
- We change the text to "Saving..." and start the save process.
- In this part of code, after all has been saved, we start the initialize with a timeout of 3 seconds.
- After the timeout we change the save to "Saved" with a timeout of 2 seconds.
- After this timeout we change the text to "Save" again.

In this way is clear that we are saving and refreshing. It uses the same two timeout than now but in different order and calling one inside the other, and we only need to add the new "Saving..." message.

Reply:
I guess 4 seconds will be enough so making them the same the buttons updates more linear.